docs/ABOUT.md

```diff
@@ -1,14 +1,31 @@
 # About
 
-<!-- TODO: write document
+[SQLite][sqlite] is an open-source lightweight, serverless, and self-contained relational database management system.
```

Comment:
I'd suggest a link to Wikipedia or some other beginner explanation of relational databases here for "relational database management system". Students not enrolled in this track will see this prominently and may not know anything about databases.